Makefile

```diff
@@ -119,8 +121,15 @@ BUILDFLAGS += -ldflags "-X $(ROOT_PACKAGE)/pkg/features.FeatureFlagToken=$(FEATU
 RELEASE_ORG_REPO := cloudbees/cloudbees-jenkins-x-distro
 endif
 
+ifdef JUNIT_REPORT
+# REPORT_PIPE must go at end of line
+OUTFILE := $(shell mktemp)
+REPORT_PIPE := -test.v 2>&1 | tee $(OUTFILE) | go-junit-report > build/integration.junit.xml & tail -f $(OUTFILE)
```

Comment:
Can we also keep the HTML report?

Reply:
I'd like to open it in the browser. The Junit XML is hard to visualise without extra tools.

Reply:
yeah, i assumed no one would use that, i'll add it back in

Reply:
Actually that's the other PR I have open ;-)